File: app/src/main/java/com/abitcreative/popularmovies/MyMoviesApplication.java
package com.abitcreative.popularmovies;

import android.app.Application;
import android.preference.PreferenceManager;

/**
 * Created by daber on 19/10/16.
 */

public class MyMoviesApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        PreferenceManager.setDefaultValues(this, R.xml.preferences, false);
    }
}

File: app/src/main/java/com/abitcreative/popularmovies/MyMoviesApplication.kt
package com.abitcreative.popularmovies

import android.app.Application
import android.preference.PreferenceManager

/**
 * Created by daber on 16/10/16.
 */

class MyMoviesApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        PreferenceManager.setDefaultValues(this, R.xml.preferences, false)
    }
}
